src/cmdlib.sh

```diff
@@ -307,7 +307,7 @@ EOF
 
     # 'pci' bus doesn't work on aarch64
     pcibus=pci.0
-    if [ "$(arch)" = "aarch64" ]; then
+    if [ "$(arch)" == "aarch64" ]; then
```

Comment:
The bash documentation says

> String operators:
> ...
> STRING1 = STRING2
>     True if the strings are equal.

while `==` is not documented.  Are you seeing incorrect behavior with this?

Reply:
The above was in reference to the `[` command specifically, I see the manual later says:

>       string1 == string2
>       string1 = string2
>              True if the strings are equal.  = should be used with the test command for POSIX conformance.